extensions/interactions/CodeRepl/CodeRepl.html

```diff
@@ -41,12 +41,6 @@
     padding: 8px;
     width: 100%;
   }
-
-  @media screen and (max-width: 959px) {
```

Comment:
Hm, not objecting to these changes (this one and the one in GraphInput.html), but could you explain why they were made?

Reply:
With these still in place, the graph or code interaction will appear below the tutor card but much fatter and it just looks awkward because there's empty space around the tutor card